chat-plugins/poll.js

```diff
@@ -62,14 +65,15 @@ class Poll {
 		this.options.forEach((option, number) => {
 			output += '<div style="margin-top: 5px"><button value="/poll vote ' + number + '" name="send" title="Vote for ' + number + '. ' + Tools.escapeHTML(option.name) + '">' + number + '. <strong>' + Tools.escapeHTML(option.name) + '</strong></button></div>';
 		});
-		output += '<div style="margin-top: 7px; padding-left: 12px"><button value="/poll results" name="send" title="View results - you will not be able to vote after viewing results"><small>(View results)</small></button></div>';
+		output += '<div style="margin-top: 7px; padding-left: 12px"><hr><button value="/poll results" name="send" title="View results - you will not be able to vote after viewing results"><small>(View results)</small></button></div>';
```

Comment:
it would kinda give space
and a line to separate the poll options and View Results Button